Formula/ntlink.rb

```diff
@@ -14,49 +14,42 @@ class Ntlink < Formula
   end
 
   depends_on "cmake" => :build
-  depends_on "gcc@11" => :build if OS.linux?
-  depends_on "meson" => :build
-  depends_on "ninja" => :build
   depends_on "abyss"
+  depends_on "btllib"
   depends_on "igraph"
   depends_on "numpy"
-  depends_on "python@3.10"
+  depends_on "python@3.8"
 
   uses_from_macos "libxml2"
   uses_from_macos "zlib"
 
-  fails_with gcc: "5" if OS.linux?
-
   def install
-    system "make", "-C", "src"
     ENV.prepend_path "PATH", libexec/"bin"
     xy = Language::Python.major_minor_version "python3"
     ENV.prepend_path "PYTHONPATH", libexec/"lib/python#{xy}/site-packages"
-    inreplace "bin/ntlink_pair.py", "/usr/bin/env python3", Formula["python@3.10"].bin/"python3.10"
-    inreplace "bin/ntlink_stitch_paths.py", "/usr/bin/env python3", Formula["python@3.10"].bin/"python3.10"
-    inreplace "bin/ntlink_overlap_sequences.py", "/usr/bin/env python3", Formula["python@3.10"].bin/"python3.10"
-    inreplace "bin/ntlink_filter_sequences.py", "/usr/bin/env python3", Formula["python@3.10"].bin/"python3.10"
-    inreplace "bin/ntlink_patch_gaps.py", "/usr/bin/env python3", Formula["python@3.10"].bin/"python3.10"
-    inreplace "ntLink", "PYTHONPATH=$(ntlink_path)/src/btllib/install/lib/btllib/python",
-    "PYTHONPATH_ntlink=$(ntlink_path)/src/btllib/install/lib/btllib/python:$(PYTHONPATH)"
-    inreplace "ntLink", "PYTHONPATH=$(PYTHONPATH)", "PYTHONPATH=$(PYTHONPATH_ntlink)"
+    inreplace "bin/ntlink_pair.py", "/usr/bin/env python3", Formula["python@3.8"].bin/"python3.8"
+    inreplace "bin/ntlink_stitch_paths.py", "/usr/bin/env python3", Formula["python@3.8"].bin/"python3.8"
+    inreplace "bin/ntlink_overlap_sequences.py", "/usr/bin/env python3", Formula["python@3.8"].bin/"python3.8"
+    inreplace "bin/ntlink_filter_sequences.py", "/usr/bin/env python3", Formula["python@3.8"].bin/"python3.8"
+    inreplace "bin/ntlink_patch_gaps.py", "/usr/bin/env python3", Formula["python@3.8"].bin/"python3.8"
+    inreplace "bin/ntlink_liftover_mappings.py", "/usr/bin/env python3", Formula["python@3.8"].bin/"python3.8"
```

Comment:
Suggest you use `rewrite_shebang detected_python_shebang, ...` for this instead. See `grep -r 'rewrite_shebang' $(brew --repo homebrew/core)` for some examples.

Reply:
Thanks for that tip @jonchang - that's much cleaner!